### PR TITLE
Add a VERSION constant

### PR DESCRIPTION
**Pull request by djberg96**
The moves VERSION out of the toplevel directory and into lib/winrm/version.rb, and modifies the gemspec and base file as needed.

### CODE REVIEW

File: winrm.gemspec [hunk 1]
```diff
@@ -1,12 +1,11 @@
 # encoding: UTF-8
 require 'date'
-
-version = File.read(File.expand_path('../VERSION', __FILE__)).strip
+require_relative 'lib/winrm/version'
```
Comment (sneal) [inline winrm.gemspec]: The build is failing with

```
There was a LoadError while loading winrm.gemspec: 
cannot infer basepath from
  /home/travis/build/WinRb/WinRM/winrm.gemspec:3:in `require_relative'
```

Might need to change this too

```
require File.expand_path('../lib/winrm/version', __FILE__)
```

File: lib/winrm/version.rb [hunk 2]
```diff
@@ -0,0 +1,4 @@
+module WinRM
+  # The version of the WinRM library
+  VERSION = '1.3.7'
```
Comment (sneal) [inline lib/winrm/version.rb]: Lets go ahead and bump the version to 1.4.0 since we're adding some new functionality, albeit minor.

**Comment by djberg96** [comment]
@sneal Ok, updatd. The current failures seem to be something unrelated:

```
1) WinRM::Output parse CLIXML errors and convert to Strings and/or Exceptions
     # Not yet implemented
     # ./spec/output_spec.rb:109
```

**Comment by sneal** [comment]
@djberg96 Looks good, the current failure are RuboCop violations in version.rb

```
lib/winrm/version.rb:1:1: C: Missing top-level module documentation comment.
module WinRM
^^^^^^
lib/winrm/version.rb:1:1: C: Missing utf-8 encoding comment.
module WinRM
```

**Comment by djberg96** [comment]
@sneal That ok?

**Comment by sneal** [comment]
@djberg96 Sorry, the changes look good minus the RuboCop violations. Those are easy enough to fix. LMK if you need help fixing those.

**Comment by djberg96** [comment]
@sneal I don't really know what you want there, since there's already a toplevel comment in winrm.rb for the WinRM module. I don't know if that would goof up the doc generators.

**Comment by sneal** [comment]
LGTM thanks!

**Comment by sneal** [comment]
@djberg96 1.4.0 released on rubygems

**Comment by djberg96** [comment]
@sneal Thanks! BTW, what else was changed for 1.4.0? I don't see an update in the changelog.

**Pull request merged**